src/character.h

```diff
@@ -1738,18 +1738,26 @@ class Character : public Creature, public visitable<Character>
         void update_stamina( int turns );
 
     protected:
-        void on_stat_change( const std::string &, int ) override {}
         void on_damage_of_type( int adjusted_damage, damage_type type, body_part bp ) override;
-        virtual void on_mutation_gain( const trait_id & ) {}
-        virtual void on_mutation_loss( const trait_id & ) {}
     public:
-        virtual void on_item_wear( const item & ) {}
-        virtual void on_item_takeoff( const item & ) {}
-        virtual void on_worn_item_washed( const item & ) {}
-
+        /** Called when an item is worn */
+        void on_item_wear( const item & );
```

Comment:
i'm pretty sure travis is gonna yell at you if the name of the variable in the declaration doesn't match the name of the variable in the definition.
```suggestion
        void on_item_wear( const item &it );
```
this goes for all of it

Reply:
Having no name is fine.  It only complains if both places have a name and they don't match.  In this case I think having a name offers no additional information so it would be fine to not have any name on the declaration (but equally, having a name does little harm).